pkg/module/template.go

```diff
@@ -48,6 +49,7 @@ spec:
 {{- end}}
   descriptor:
 {{yaml .Descriptor | printf "%s" | indent 4}}
+  mandatory: {{.Mandatory}}
```

Comment:
I suggest moving the Mandatory field right after Channel (before data and descriptor) so that it can be easily found in the ModuleTemplate